heimdall

```diff
@@ -79,6 +95,64 @@ case ${args[0]} in
 
         # If the target param contains a username, split it out so we can determine the host dns.
         host=${args[0]}
+        if [[ ${host} =~ "#" ]]
+        then
+            SEARCH_SERVICE=$(echo ${host} | cut -f1 -d\#)
+            SEARCH_CLUSTER=$(echo ${host} | cut -f2 -d\#)
```

Comment:
Awesome, just awesome. My one gripe is the order is backwards. I had to dissect this to figure out why `cluster#service` wasn't working. I think using `service#cluster` is backwards. I'm OK with either direction though, this is a good addition.

Reply:
I thought about your comment extensively, and decided to leave it as is. For me, it's the email syntax. [service]@[cluster]. Since @ is already taken, i went one to the right :)